src/common/catalog-entities/general.ts

```diff
@@ -39,6 +39,10 @@ export class GeneralEntity extends CatalogEntity<CatalogEntityMetadata, CatalogE
     navigate(this.spec.path);
   }
 
+  onClickDetailIcon() {
```

Comment:
Still don't like this name.

Reply:
Just rebased to master, no new commits yet :)